### PR TITLE
Readme corrections, use port 30555 for backend everywhere

### PR DESCRIPTION
**Pull request by a-roberts**
Be consistent everywhere

### CODE REVIEW

File: backend/README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ docker build -t backend:v1.0.0 .
 docker run -d -p 27017:27017 mongo
 docker inspect <the mongo container name> | grep IPAddress
 export MONGO_URL=the IP address from above>
```
[the remark this answers is not in the document]
Comment (a-roberts) [reply backend/README.md]: Good catch, thanks!

**Comment by a-roberts** [comment]
I noticed a few other corrections and so updated this branch accordingly, @BethGriggs please take another look, thanks

File: backend/README.md [hunk 2]
```diff
@@ -10,9 +10,8 @@ Install Docker on your Mac.
 ```
 docker build -t backend:v1.0.0 .
 docker run -d -p 27017:27017 mongo
```
Comment (BethGriggs) [inline backend/README.md]: Should this now be mern-mongo to match below line

Comment (a-roberts) [reply backend/README.md]: Yes you're right, I've added it

**Pull request merged**